src/librustdoc/externalfiles.rs

```diff
@@ -70,17 +70,14 @@ pub fn load_string<P: AsRef<Path>>(file_path: P) -> Result<String, LoadStringErr
     let result = File::open(file_path)
                       .and_then(|mut f| f.read_to_end(&mut contents));
     if let Err(e) = result {
-        let _ = writeln!(&mut io::stderr(),
-                         "error reading `{}`: {}",
-                         file_path.display(), e);
+        let _ = eprintln!("error reading `{}`: {}", file_path.display(), e);
         return Err(LoadStringError::ReadFail);
     }
     match str::from_utf8(&contents) {
         Ok(s) => Ok(s.to_string()),
         Err(_) => {
-            let _ = writeln!(&mut io::stderr(),
-                             "error reading `{}`: not UTF-8",
-                             file_path.display());
+            let _ = eprintln!("error reading `{}`: not UTF-8",
+                              file_path.display());
```

Comment:
Remove the `let _ = `. (Same for a few lines above)

Also, the `use std::io;` is no longer needed.

```
[00:34:44] error: unused import: `std::io`
[00:34:44]   --> /checkout/src/librustdoc/externalfiles.rs:13:5
[00:34:44]    |
[00:34:44] 13 | use std::io;
[00:34:44]    |     ^^^^^^^
[00:34:44]    |
```